circle.yml

```diff
@@ -72,14 +72,14 @@ dependencies:
         dashboard_workers: 6
         skip_seed_all: true" >> locals.yml
     - |
-        case $CIRCLE_NODE_INDEX in 0) echo "Skipping" ;; *) echo "
+        echo "
         use_dynamo_tables: false
         use_dynamo_properties: false
         no_https_store: true
         override_dashboard: \"localhost-studio.code.org\"
         override_pegasus: \"localhost.code.org\"
         dashboard_port: 3000
-        pegasus_port: 3000" >> locals.yml ;; esac
+        pegasus_port: 3000" >> locals.yml
```

Comment:
We could technically merge this step with the one above it, but leaving it separate for now so it's easy to undo in the future if we'd like.